lib/screens/calc_screen.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class CalcScreen extends StatelessWidget {
+  const CalcScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Column(
+        children: [
+          Center(
+            child: ElevatedButton.icon(
+              onPressed: () {},
+              icon: Image.asset('assets/dark_mode.png'),
```

Comment:
nazwę assetu fajnie mieć jest wyciągniętą do innego pliku.
Plik możesz nazwać constants.dart i zrobić klasę Constants (jezeli jest taka dostępna, jeśli nie to jakąś inną nazwę). Struktura powinna wyglądać coś jak to:
`class Constants {
   Constants._();

   // Assets
   static const String darkModeAssetPath = 'assets/dark_mode.png';

   // Dimens  (tutaj dodajesz jakieś szerokości czy wysokości dla różnych widgetów
  // czyli tutaj np. static const int widthButton = 30;
`

Reply:
i taki pliczek wrzucasz to folderu utils, umieszczasz go w folderze 'lib'